modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -118,7 +118,7 @@ This format accepts properties that provide instructions for reading input data.
 [cols="1,1,1,3a", options="header"]
 |===
 |Parameter |Type |Default|Description
-|`schemaUrl` (Required) |`String`|`null`|The URL for the Avro schema. Valid URI schemes are `classpath://`, `file://`, or `http://`.
+|`schemaUrl` (Required) |`String`|`''`|The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
```

Comment:
```suggestion
|`schemaUrl` (Required) |`String`|`''`|The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`. This property is optional and defaults to the schema embedded in the input Avro file.schemaUrl reader property defaults to the schema embedded in the input Avro file. DataWeave requires an embedded schema.
```

Reply:
TODO: add to source

Reply:
added to master

Reply:
added this: "added this to master bc the description is used by both reader and writer:

"The URL for the Avro schema. Valid schemes are `classpath://`, `file://`, or `http://`. For the reader, this property is optional but defaults to the schema embedded in the input Avro file. The reader requires an embedded schema. For the writer, DataWeave requires a schema value.""